src/Microsoft.Diagnostics.Monitoring.WebApi/Stacks/NameFormatter.cs

```diff
@@ -20,24 +20,31 @@ internal sealed class NameFormatter
         private const string ArrayType = "_ArrayType_";
         private const string CompositeType = "_CompositeType_";
 
+        private const char DotSeparator = '.';
         private const char NestedSeparator = '+';
         private const char GenericStart = '[';
         private const char GenericSeparator = ',';
         private const char GenericEnd = ']';
 
+        internal enum TypeFormat
```

Comment:
I'm not certain that we'll have some other format, so maybe change this to be a `bool` in the parameters e.g. `bool useSimpleName`.

Reply:
Unless you feel strongly about it, I think I'd rather use the enum - I initially prototyped this using a bool, but I feel like this gives us a lot more future flexibility (without refactoring everything) without introducing much more verbosity. In the most recent change I did update this to use `Full` and `Simple` though.